request_a_govuk_domain/request/tasks.py

```diff
@@ -162,3 +167,38 @@ def check_email_failure_and_notify() -> None:
                     # Delete the notification response id, as the necessary action after email failure email has been
                     # taken, so no need to track anymore
                     notification_response_id.delete()
+
+
+@shared_task
+@transaction.atomic
+def check_application_status() -> None:
+    """
+    Checks appliction status periodically
+    """
+    time_flag = TimeFlag.objects.first()
+
+    if time_flag:
+        more_info_application = Application.objects.filter(status="more_information")
```

Comment:
Can we use value_list(..) so we only load those fields we require? Otherwise it wil load the whole object graph/

Reply:
Also we can do the time comparison as part of the query?